crates/bevy_window/src/window.rs

```diff
@@ -255,6 +255,27 @@ impl Default for Window {
 }
 
 impl Window {
+    /// Gets the window's current maximized state.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web:** Unsupported.
+    pub fn is_maximized(&self) -> bool {
```

Comment:
What happens on unsupported platforms?
Should there be conditional compilation for it?
Maybe we could return an Option<bool>, to be consistent with is_minimized and return None on unsupported platforms?

Reply:
Yeah, this is a good call. Either approach should work, although I haven't seen many examples of conditional compilation for this kind of stuff in the codebase (I don't know much of it though) I think it sounds sensible. What I also think is the window implementation caters too much to winit in some aspects so maybe in a future it can be a bit more "decoupled" and we'd have an API with almost 0 regards of the backend.

I think for now we could go with the optional approach, which will pair well with the other method. Thanks for calling this out.